infobloxddi_connector.py

```diff
@@ -44,6 +46,27 @@
 # List containing HTTP codes to be considered as success
 SUCCESS_RESPONSE_CODES = [consts.INFOBLOX_REST_RESP_SUCCESS, consts.INFOBLOX_REST_RESP_CREATE_SUCCESS]
 
+# We can pick FIPS cipers for TLS to work.
+# We need to prevent the clients to ask to speak SHA1
+CIPHERS = (
```

Comment:
More info on the fix can be found at https://stackoverflow.com/questions/40373115/how-to-select-specific-the-cipher-while-sending-request-via-python-request-modul

Reply:
![image](https://user-images.githubusercontent.com/82124762/153104417-0ee39cb1-1919-409f-87e6-b257d84e16f1.png)